tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskCliBasicUsageTests.scala

```diff
@@ -390,14 +391,16 @@ class WskCliBasicUsageTests extends TestHelpers with WskTestHelpers {
       val memoryLimit = 512 MB
       val logLimit = 1 MB
       val timeLimit = 60 seconds
+      val concurrencyLimit = 500
 
       assetHelper.withCleaner(wsk.action, name) { (action, _) =>
         action.create(
           name,
           Some(TestUtils.getTestActionFilename("helloAsync.js")),
           memory = Some(memoryLimit),
           timeout = Some(timeLimit),
-          logsize = Some(logLimit))
+          logsize = Some(logLimit),
+          concurrency = Some(concurrencyLimit))
```

Comment:
also need to add the expected concurrency value here..?
https://github.com/apache/incubator-openwhisk-cli/blob/0640ecafb67b37f489665c8d46ab054b94a8a763/tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskCliBasicUsageTests.scala#L414

Reply:
yes! thanks